src/services/task.service.ts

```diff
@@ -0,0 +1,36 @@
+import { AccessToken, APIResponse } from '../types';
+import { Task } from '../types';
+
+import { APIService } from './api-service';
+
+export class TaskService extends APIService {
+  private getAuthHeaders(userAccessToken: AccessToken) {
```

Comment:
Think this can be made part of `APIService`. It can be then reused.

Reply:
Implemented.